package.json

```diff
@@ -22,7 +22,7 @@
     "dev": "pnpm run build:test --watch",
     "eslint": "eslint src && eslint test",
     "eslint:fix": "eslint --fix src && eslint --fix test",
-    "prepare": "ts-patch install && typia patch && git update-index --assume-unchanged .env",
+    "prepare": "ts-patch install && typia patch && git update-index --assume-unchanged .env && npm run list-connectors",
```

Comment:
can you fix the command to `pnpm`?

Reply:
when you use npm command on pnpm, it is executed as pnpm so fine